core/src/components/modal/gestures/swipe-to-close.ts

```diff
@@ -39,16 +39,18 @@ export const createSwipeToCloseGesture = (
   };
 
   const onMove = (detail: GestureDetail) => {
-    const step = detail.deltaY / height;
-    if (step < 0) { return; }
+    const step = clamp(0.0001, detail.deltaY / height, 0.9999);
 
     animation.progressStep(step);
   };
 
   const onEnd = (detail: GestureDetail) => {
-    const velocity = detail.velocityY;
-    const step = detail.deltaY / height;
-    if (step < 0) { return; }
+    let velocity = detail.velocityY;
+    if (velocity <= 0) {
+      velocity = 0.0001; // Fixes modal stuck when swiping up from the bottom very fast
+    }
```

Comment:
```suggestion
    const velocity = detail.velocityY;
```

I don't think we need to clamp the velocity. Swiping is working fine for me with just the `clamp`.

Reply:
You were right. I removed it and pushed a new version.